x-pack/test/spaces_api_integration/apis/saved_objects/lib/space_test_utils.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function getUrlPrefix(urlContext) {
+  return urlContext ? `/s/${urlContext}` : ``;
+}
+
+// Spaces do not actually prefix the ID, but this simplifies testing positive and negative flows.
```

Comment:
Sorry for being dense, would you mind elaborating on this?

Reply:
This comment will be outdated by https://github.com/legrego/kibana/pull/5, but here, I am prefixing each object id with its space id inside the es archive. This allows me to reuse the base id, so that we have less hard-coded ids within tests, while still letting me assert that I am returning the expected objects, and only the expected objects.